Networking/NetworkingTests/Mapper/ListMapperTests.swift

```diff
@@ -0,0 +1,112 @@
+import Testing
+@testable import Networking
+
+struct ListMapperTests {
```

Comment:
❓ How about adding test case(s) for the mapper use case in https://github.com/woocommerce/woocommerce-ios/pull/15441, replacing `ProductListMapper`? There are already a few test cases for `ProductsRemote.loadProductsForPointOfSale`, but not with the expectations on properties.

Reply:
Hmm. Most of those seem like they're intended to test the Remote, not the mapper. One or two are testing the mapper by default, so I could move those, but they have predicates that suggest they're _meant_ to test the remote. 

For example `test_loadProductsForPointOfSale_when_page_has_products_then_loads_expected_products` is only testing mapper code, but the mention of pages suggests it's supposed to test whether the correct page is loaded.

I'll move that anyway, and the one about empty data, but the majority of it is testing whether the remote uses the right parameters, etc.

Reply:
I looked at doing this and it seems like a duplicative test – we're already checking 3 types work, that seems like enough in the ListMapper tests. I'll add the empty data case though.